src/utils.tsx

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+
+function makeid(length: number) {
+  let result = '';
+  let characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  let charactersLength = characters.length;
+  for (let i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+}
+
+function getCurrentSection(id: string): string {
+  return id.split('_')[1];
+}
+
+function getCurrentField(id: string): string {
+  return _.last(id.split('_')) || '';
```

Comment:
No need to lodash here, the built-in `pop` method should do the trick.

```suggestion
  return id.split('_').pop();
```

Furthermore, you have to notice that if the split does not work, the whole id will be returned and not `''`